hosts_example.yml

```
@@ -315,6 +315,21 @@ kafka_broker:
       #   broker.rack: us-east-1d
       #   replica.selector.class: org.apache.kafka.common.replica.RackAwareReplicaSelector
 
+      # Per host, in case there are multiple interfaces and you need to listen on only one of them,
+      # you can also specify which interface to listen on, per listener.
+      # On each listener, you need to add an 'ip' value. This will adjust the 'listeners' section per host.
+      #
+      # kafka_broker_custom_listeners:
+      #   broker:
+      #     ip: 123.123.123.123
+      #     hostname: 'broker1.something.local'
+      #   internal:
+      #     ip: 123.123.123.123
+      #     hostname: 'broker1.something.local'
+      #   client_listener:
+      #     ip: 123.123.123.123
+      #     hostname: 'broker1.something.local'
```

```suggestion
      #     hostname: broker1.something.local
      #   internal:
      #     ip: 123.123.123.123
      #     hostname: broker1.something.local
      #   client_listener:
      #     ip: 123.123.123.123
      #     hostname: broker1.something.local
```

these quotes arent necessary right?

---

no, you're right... removed